src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -75,11 +83,22 @@ trait ClusterComponent extends LeoComponent {
       }
     }
 
-    def deleteCluster(googleId: UUID):DBIO[Int] = {
-       clusterQuery.filter(_.googleId === googleId)
-         .map(c => (c.destroyedDate, c.status))
-         .update((Option(Timestamp.from(java.time.Instant.now())), ClusterStatus.Deleting.toString))
+    def deleteCluster(googleId: UUID): DBIO[Int] = {
```

Comment:
suggest renaming this to `markPendingDeletion` as i'm repeatedly thrown off every time i read it

Reply:
Yeah makes sense, goes well with `completeDeletion`